ion-schema/src/isl/isl_constraint.rs

```diff
@@ -446,12 +447,9 @@ impl IslConstraint {
                 RangeType::NonNegativeInteger,
             )?)),
             "valid_values" => Ok(IslConstraint::ValidValues(value.try_into()?)),
-            _ => Err(invalid_schema_error_raw(
-                "Type: ".to_owned()
-                    + type_name
-                    + " can not be built as constraint: "
-                    + constraint_name
-                    + " does not exist",
+            _ => Ok(IslConstraint::Unknown(
```

Comment:
Is this going to match against `name`? I believe a name should already be captured elsewhere, so it should not be captured here, but can you double check that?